packages/desktop/src/renderer/app/components/context-menu/context-menus.ts

```diff
@@ -87,6 +87,16 @@ export const FoldersContextMenu = (folderUUID: string): ContextMenuItem[] => [
     icon: 'folder',
     disabled: false
   },
+  {
+    payload: {
+      subject: 'folder',
+      action: 'toggle',
+      subjectUUID: folderUUID
+    },
+    label: 'Toggle folder',
```

Comment:
Could you edit the label for something more precise, like:

```suggestion
    label: 'Toggle direct child routes',
```